### ISSUE TITLE
workflow tables cleanup

### ISSUE THREAD

**Issue by niphlod** [rundeck/rundeck#2630]
Issue type: Bug report

**My Rundeck detail**

* Rundeck version: 2.8.3
* install type: launcher
* OS Name/version: WS2012R2
* DB Type/version: mssql

**Expected Behavior**
Cleaning up executions correctly cleans up every detail associated with that execution from the database

**Actual Behavior**
When cleaning up executions, the base_report and the execution table are correctly cleaned. 
workflow, workflow_step and workflow_workflow_step instead aren't cleaned up.

-----

To sum up in an example, I'd expect this query to return 0 records
```
SELECT id
  FROM workflow
 where id not in (
 select workflow_id from execution
 )
```
Instead, I have in it basically every record since installation time.

**Comment by niphlod** [comment]
whoops, probably a duplicate of https://github.com/rundeck/rundeck/issues/1654 .

BTW, dunno if you wanna take care of it in the code or in the db model. It seems that FK are created without CASCADE DELETE on it.

**Comment by ahonor** [comment]
duplicate of #1654 .
